Python/Product/Common/Strings.resx

```diff
@@ -2968,6 +2968,14 @@ Please specify at least one package.</value>
     <value>Results for pytest not found {0}</value>
     <comment>{0} is the expected file path</comment>
   </data>
-
-    
+  <data name="PythonVersionNotSupportedDontShowMessageAgain" xml:space="preserve">
+    <value>Don't show this message again</value>
+  </data>
+  <data name="PythonVersionNotSupportInfoBarText" xml:space="preserve">
+    <value>You are using Python {0} and some features might not work because VS doesn't support it yet.</value>
```

Comment:
UX folks were okay with abbreviating VS?

Reply:
Should rename to PythonVersionNotSupport*ed*InfoBarText

Reply:
Recommendation is: "You're using Python 3.8 and some features might not work.", but it fails to convey the reason why it won't work, so I added that part in. Changed VS to "Visual Studio (see next commit).